doc/user_doc/vic_admin/upgrade_vch_options.md

```diff
@@ -2,6 +2,8 @@
 
 The command line utility for vSphere Integrated Containers Engine, `vic-machine`, provides an `upgrade` command that allows you to upgrade virtual container hosts to a newer version. The options that `vic-machine upgrade` requires depend on the location in your vSphere environment in which you deployed the virtual container host to upgrade.
 
+**IMPORTANT**: Due to the substantial changes in vSphere Integrated Containers version 0.7, you cannot use `vic-machine upgrade` to upgrade from version 0.6 to version 0.7. You can use `vic-machine upgrade` to upgrade from more recent builds to version 0.7.
```

Comment:
Maybe we can keep the statement for now for the open source users, but remove it before we publish the GA docs? I used upgrade this morning from a very recent build to 0.7 and it worked fine. I assume that we definitely don't want to encourage people to upgrade a pre-GA release to GA and then use it in production...